examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -47,24 +49,24 @@
         verticalalignment='top',
         transform=ax.transAxes)
 
-ax.text(left, 0.5*(bottom + top), 'right center',
+ax.text(left, 0.5 * (bottom + top), 'right center',
```

Comment:
I'm -1 on this and the following changes. Note that PEP8 AFAIK does not recommend to add whitespace before and after this operator.